tools/skyportal_query_examples.ipynb

```diff
@@ -0,0 +1,252 @@
+   "source": [
+    "api_token = \"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "14515539",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "headers = {'Authorization': f'token {api_token}'}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "56f96c9c",
+   "metadata": {},
+   "source": [
+    "# Input endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3714175e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = \"telescope\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d074ed59",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "url = urllib.parse.urljoin(host, f'/api/{endpoint}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91ac58de",
+   "metadata": {},
+   "source": [
+    "# Post Example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6ddd8634",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "post_data = {\n",
+    "    'name': \"My Telescope\",\n",
+    "    'nickname': \"my_tele\",\n",
+    "    'lat': 30.0,\n",
+    "    'lon': 60.0,\n",
+    "    'elevation': 1000.0,\n",
+    "    'diameter': 10.0,\n",
+    "    'robotic': True,\n",
+    "}\n",
+    "r = requests.post(url, headers=headers, json=post_data)\n",
+    "print(r.json())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "947d5e25",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "post_data = {\n",
+    "    'name': \"My Other Telescope\",\n",
+    "    'nickname': \"other_tele\",\n",
+    "    'lat': -30.0,\n",
+    "    'lon': -60.0,\n",
+    "    'elevation': 1000.0,\n",
+    "    'diameter': 10.0,\n",
+    "    'robotic': True,\n",
+    "}\n",
+    "r = requests.post(url, headers=headers, json=post_data)\n",
+    "print(r.json())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "40b4ceb2",
+   "metadata": {},
+   "source": [
+    "# Get Example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f2cfde10",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r = requests.get(url, headers=headers)\n",
+    "print(r.json())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "31f17230",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "params = {\n",
+    "    \"latitudeMin\": -45.0,\n",
+    "    \"latitudeMax\": -15.0,\n",
+    "    \"longitudeMin\": -75.0,\n",
+    "    \"longitudeMax\": -45.0,\n",
+    "}\n",
+    "\n",
+    "r = requests.get(url, headers=headers, params=params)\n",
+    "print(r.json())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "83390881",
+   "metadata": {},
+   "source": [
+    "# Delete Example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0d8bc15c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "telescope_id = 19"
```

Comment:
Maybe want to link this number to the ID from the GET example?

Reply:
Yup fixed.